docs/spec/amp-errors.md

```diff
@@ -10,23 +10,37 @@ the `betaErrorReportingUrl` config. E.g. for `cdn.ampproject.org`
 The following fields are reported:
 
 -   `v=string` - AMP version
+-   `noamp={0,1}` - whether the document contains non-AMP JavaScript
```

Comment:
was this just missed documentation before? (i do see in the testing code below, but just double checking)

Reply:
Yeah, looks like we kept adding fields in `error-reporting.js` but rarely updated `amp-errors.md` to correspond. I wish we had something like go/ifttt for GitHub 😅